R/arx_forecaster.R

```diff
@@ -171,7 +171,9 @@ arx_fcast_epi_workflow <- function(
     step_epi_ahead(!!outcome, ahead = args_list$ahead)
   r <- r %>%
     step_epi_naomit() %>%
-    step_training_window(n_recent = args_list$n_training)
+    step_training_window(n_recent = args_list$n_training) %>%
+    check_enough_train_data(all_predictors(), n = args_list$check_enough_data_n, skip = FALSE)
+
   if (!is.null(args_list$check_enough_data_n)) {
```

Comment:
How come this is separate from the `check_enough_train_data` below?

Reply:
the short of it is I didn't read closely enough to notice that that was included.